packages/components/src/components/block-section/block-section.tsx

```diff
@@ -76,7 +76,7 @@ export class BlockSection extends LitElement {
   /**
    * When `true`, expands the component and its contents.
    *
-   * @deprecated Use `expanded` prop instead.
+   * @deprecated in v3.1.0, removal target v6.0 - Use `expanded` prop instead.
```

Comment:
Similar to above consistency:
```suggestion
   * @deprecated in v3.1.0, removal target v6.0.0 - Use the `expanded` property instead.
```